webserver.py

```diff
@@ -91,30 +91,37 @@ def save(self, slug):
                                [slug, json_game_state, json_game_state])
 
     def initialize(self):
-        for seed_id in GAME_CONFIG['seeds']:
-            if self.data['seedCounts'].get(seed_id) is None:
-                self.data['seedCounts'][seed_id] = 0
         for resource in GAME_CONFIG['starting_resources']:
             if self.data['resources'].get(resource) is None:
-                self.data['resources'][resource] = GAME_CONFIG['starting_resources'][resource]
+                if GAME_CONFIG['starting_resources'][resource] > 0:
+                    self.data['resources'][resource] = GAME_CONFIG['starting_resources'][resource]
+        if self.data.get('plots') is None:  # temp
+            self.data['plots'] = {}
+        if 'unlockCount' in self.data:      # temp
+            del self.data['unlockCount']
         for i in range(GAME_CONFIG['field_width']):
             for j in range(GAME_CONFIG['field_height']):
-                if self.data.get('plot' + str(i) + '_' + str(j)) is None:
-                    self.data['plot' + str(i) + '_' + str(j)] = {'seedType': 0, 'sowTime': 0, 'locked': 1}
-        for i in range(2):
-            for j in range(2):
-                self.get_plot(i, j)['locked'] = 0
+                if 'plot' + str(i) + '_' + str(j) in self.data:
+                    if self.data['plot' + str(i) + '_' + str(j)]['locked'] == 0:
+                        self.data['plots'][str(i) + '_' + str(j)]['seedType'] = self.data['plot' + str(i) + '_' + str(j)]['seedType']
+                        self.data['plots'][str(i) + '_' + str(j)]['sowTime'] = self.data['plot' + str(i) + '_' + str(j)]['sowTime']
+                    del self.data['plot' + str(i) + '_' + str(j)]
+        for i in range(GAME_CONFIG['starting_field_width']):
+            for j in range(GAME_CONFIG['starting_field_height']):
+                if self.data['plots'].get(str(i) + '_' + str(j)) is None:
+                    self.data['plots'][str(i) + '_' + str(j)] = {'seedType': 0, 'sowTime': 0}
         return self
 
     def check_cash(self):
```

Comment:
This can be simplified a bit by getting rid of both `total_*` variables.  As soon as either of them is >0, you can `return` and abandon the method immediately.  If you get to the end without bailing out, you know you're in a situation where you need to give them a bit more cash